images/bridge2/ui/session.html

```diff
@@ -13,12 +13,20 @@
   <script type="module">
 
 import {ReflectCommands} from "./ui/substrate-r0.js";
+import {CommandPanel} from "./ui/command-panel.js";
 import {Session} from "./ui/session.js";
 import {Sidebar} from "./ui/sidebar.js";
 import {Topbar} from "./ui/topbar.js";
 import {Session as SFU} from "./webrtc/session.js";
 
-window.substrate.r0.commands = new ReflectCommands();
+window.customElements.define('command-panel', CommandPanel.element);
+
+let commands = new ReflectCommands();
+window.substrate.r0.commands = commands;
+commands.index.then(() => {
+  console.log('index loaded')
+  document.querySelector('command-panel').commands = commands;
+});
```

Comment:
I'll start to test out eliminating this boiler-plate. Svelte has an option to automatically register the custom element, and we should be able to default to looking in `substrate.r0` for commands, so this may not be needed.